profiles/arch/arm/armv7a/use.force

```diff
@@ -2,3 +2,14 @@
 armv5te
 armv6
 armv6t2
+
+cpu_flags_arm_v4
+cpu_flags_arm_v5
+cpu_flags_arm_v6
+cpu_flags_arm_v7
+
+cpu_flags_arm_edsp
+cpu_flags_arm_thumb
+cpu_flags_arm_thumb2
+cpu_flags_arm_vfp
+cpu_flags_arm_vfpv3
```

Comment:
VFPv3 is an optional extension to the ARM, Thumb® and ThumbEE instruction sets in the ARMv7-A and ARMv7-R profiles.